adafruit_ads1x15/ads1x15.py

```diff
@@ -83,7 +106,18 @@ def __init__(
         self.gain = gain
         self.data_rate = self._data_rate_default() if data_rate is None else data_rate
         self.mode = mode
+        self.comparator_queue_length = comparator_queue_length
         self.i2c_device = I2CDevice(i2c, address)
+        self.comparator_low_threshold = (
+            self._comp_low_thres_default()
+            if comparator_low_threshold is None
+            else comparator_low_threshold
+        )
+        self.comparator_high_threshold = (
+            self._comp_high_thres_default()
+            if comparator_high_threshold is None
+            else comparator_high_threshold
+        )
```

Comment:
The default is the same for both because the lower four bits are 1s in the high threshold.

```suggestion
        self.comparator_low_threshold = comparator_low_threshold
        self.comparator_high_threshold = comparator_high_threshold
```

Reply:
You are correct. I mistakenly went to some effort to separate them when I didn't need to.